tla-types/src/main/scala/at/forsyte/apalache/tla/typecheck/etc/ToEtcExpr.scala

```diff
@@ -782,6 +796,34 @@ class ToEtcExpr(annotationStore: AnnotationStore, varPool: TypeVarPool) extends
         val opsig = OperT1(Seq(FunT1(IntT1(), a), IntT1()), SeqT1(a))
         mkExRefApp(opsig, Seq(fun, len))
 
+      case OperEx(BmcOper.assign, lhs, rhs) =>
```

Comment:
So are these expressions we missed in previous implementations? Are we sure we're supporting them all know?

Any tests we could do to ensure we have complete coverage?

Reply:
Now we are supporting them. There are tests for the newly added operators. As for coverage, they popped up in the integration tests.